hack/gen-swagger-doc/run-gen-swagger-docs.sh

```diff
@@ -18,7 +18,7 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../..
```

Comment:
This fixes part of that. We also need a one-line change in versionized.sh as well. I think I should make that one-line change another PR, since that not a cherry-pick. WDYT?